style.css

```diff
@@ -0,0 +1,247 @@
+*{
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+input, button {
+  font-family: inherit;
+}
+
+body {
+  background: url(img/Background.png) no-repeat;
+  background-size: cover;
+  font-family: 'Roboto', sans-serif;
+  margin: 0 auto;
+  min-width: 320px; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5838315)